yb-voyager/cmd/assessMigrationCommand.go

```diff
@@ -1400,3 +1410,27 @@ func validateAssessmentMetadataDirFlag() {
 		}
 	}
 }
+
+func validateAndSetTargetDbVersionFlag() error {
+	if targetDbVersionStrFlag == "" {
+		targetDbVersion = version.LatestStable
+		utils.PrintAndLog("Defaulting to latest stable YugabyteDB version: %s", targetDbVersion)
+		return nil
+	}
+	var err error
+	targetDbVersion, err = version.NewYBVersion(targetDbVersionStrFlag)
+
+	if !errors.Is(err, version.ErrUnsupportedSeries) {
+		return err
+	}
```

Comment:
wondering, should we explicitly add condition here for `err == nil` being handled also? Just to make the code more readable.

Reply:
Makes sense, done!